tests/lib/rules/order-in-components.js

```diff
@@ -345,6 +355,33 @@ eslintTester.run('order-in-components', rule, {
         },
       ],
     },
+    {
+      code: `export default Component.extend({
+        actions: {},
+
+        role: ${`${'sloth'}`},
```

Comment:
You could also use backslash `\` to escape the first/last template literals. Up to you though.